definitions/palabritas.md

```diff
@@ -353,10 +396,34 @@ You can use an arbitrary amount of parameters, they will be passed to the runtim
 It's up to the runtime how to interpret `0.3` in this case, and cuentitos will just pass it through.
 You can always use variables as a way to communicate with the runtime and check for types in compile time.
 
+Another way to communicate with your runtime is by using a `tag`. This serves as a marker that you can place on any line.
+
+```cuentitos
+# second_day_happy
+  * Shelter the dog.
+  tag important_decision
+```
+
+The runtime will receive a `tag` named `important_decision` so the runtime can, for example, warn the player about the consequences of taking this path.
```

Comment:
We should change the runtime for the game engine or something across the text, because we have our runtime and I'm worried it will be confusing.